gnd/src/main/java/com/google/android/gnd/ui/home/HomeScreenFragment.java

```diff
@@ -108,18 +113,45 @@ public void onCreate(@Nullable Bundle savedInstanceState) {
 
     mapContainerViewModel = getViewModel(MapContainerViewModel.class);
     projectSelectorViewModel = getViewModel(ProjectSelectorViewModel.class);
+    featureSelectorViewModel = getViewModel(FeatureSelectorViewModel.class);
+
+    featureSelectorDialogFragment = new FeatureSelectorFragment(featureSelectorViewModel);
 
     viewModel = getViewModel(HomeScreenViewModel.class);
     viewModel.getProjectLoadingState().observe(this, this::onActiveProjectChange);
     viewModel
         .getShowAddFeatureDialogRequests()
         .observe(this, e -> e.ifUnhandled(this::onShowAddFeatureDialogRequest));
     viewModel.getBottomSheetState().observe(this, this::onBottomSheetStateChange);
+    viewModel
+        .getOverlappingFeatures()
+        .as(autoDisposable(this))
+        .subscribe(this::onTapOverlappingFeatures);
     viewModel.getOpenDrawerRequests().observe(this, e -> e.ifUnhandled(this::openDrawer));
     viewModel.getAddFeatureResults().observe(this, this::onFeatureAdded);
     viewModel.getUpdateFeatureResults().observe(this, this::onFeatureUpdated);
     viewModel.getDeleteFeatureResults().observe(this, this::onFeatureDeleted);
     viewModel.getErrors().observe(this, this::onError);
+    featureSelectorViewModel
+        .getFeatureSelections()
+        .as(autoDisposable(this))
+        .subscribe(this::onFeatureSelection);
+  }
+
+  private void onTapOverlappingFeatures(ImmutableList<Feature> features) {
```

Comment:
Modern mobile UX uses "tap", but Android uses "click". To be internally consistent I'd suggest "click".